Turkey/Cleaner.cs

```diff
@@ -36,7 +36,9 @@ public static IEnumerable<string> LocalProjectCruft()
             yield return "project.lock.json";
         }
 
-        public async Task CleanProjectLocalDotNetCruftAsync()
+#pragma warning disable CA1822 // Mark members as static
```

Comment:
Similar to https://github.com/redhat-developer/dotnet-bunny/pull/91/files#r1888234678, I'm not used to seeing this message though I often don't add the `static` keyword to members that are not using instance members. I wonder if we have some specific analyser settings that trigger this?